packages/wrangler/src/secret/secretKey.ts

```diff
@@ -0,0 +1,60 @@
+import { confirm, prompt, select } from "../dialogs";
+import { readFromStdin, trimTrailingWhitespace } from "./util";
+import type { SecretKeyBody } from "./types";
+
+/**
+ * Walks the user through a wizard, configuring the secret key and building a payload
+ * from among the many permutations of types, algorithms, usages, and formats.
+ *
+ * @param name
+ * @param isInteractive
+ * @returns
+ */
+export const secretKeyWizard = async (
+	name: string,
+	isInteractive: boolean
+): Promise<SecretKeyBody> => {
+	// TODO: Take in the type of secret, 'raw', 'pkcs8', 'spki', 'jwk' and walk through different wizard
```

Comment:
There are many ways to import a cryptographic secret in the CryptoKey API. Wondering if I should stick to `raw` secrets for now, since supporting them all in one go will be a lot of work.